sbt_common/display/src/test/scala/uk/ac/wellcome/display/models/DisplaySerialisationTestBase.scala

```diff
@@ -204,6 +208,22 @@ trait DisplaySerialisationTestBase { this: Suite =>
        |}
      """.stripMargin
 
+  def contributors(c: List[Contributor[Displayable[AbstractAgent]]]) =
```

Comment:
nit: not a fan of single-character variable names as arguments, prefer slightly more descriptive, e.g. `con`, `prod`, `event`

Doubly so when it's a character like `l`!